crates/aptos/src/common/utils.rs

```diff
@@ -190,3 +200,68 @@ pub async fn get_sequence_number(
     let account = account_response.inner();
     Ok(account.sequence_number)
 }
+
+/// Error message for parsing a map
+const PARSE_MAP_SYNTAX_MSG: &str = "Invalid syntax for map.  Example: Name=Value,Name2=Value";
```

Comment:
nit: Why double space? 😄 `map.  Example`?

Reply:
Cause that's how I was taught as a kid to write sentences :P